### PR TITLE
[FA] 04 and 04-1

### PR DESCRIPTION
**Pull request by erfanMhi**

### CODE REVIEW

File: docs/fa/week04/04-1.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
+---
+lang-ref: ch.04-1
+lecturer: Alfredo Canziani
+title: Linear Algebra and Convolutions
```
Comment (Atcold) [inline docs/fa/week04/04-1.md:4]: The title needs translation.

File: docs/fa/week04/04.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+lang-ref: ch.04
+title: Week 4
```
Comment (Atcold) [inline docs/fa/week04/04.md:3]: The title needs translation.

**Review by Atcold — CHANGES_REQUESTED** [review]
Great! Just translate the titles, and we're good here!

**Comment by erfanMhi** [comment]
Dear Dr. Canziani,

The titles of the aforementioned Readmes have been translated to Farsi/Persian, and they are now ready to be published.

**Comment by erfanMhi** [comment]
Just to point out that the title of '03-3.md' Readme file hasn't been translated into Farsi too.  I wonder whether it would be ok to translate the title of this file which actually belong to week03 and update my pull request?

**Review by Atcold — APPROVED** [review]

**Pull request merged**